pkg/internal/apis/config/validate.go

```diff
@@ -101,10 +103,18 @@ func (n *Node) Validate() error {
 		}
 	}
 
+	// validate HostPath
```

Comment:
I would not expect `Validate()` to mutate any fields or depend on the host.

can we take this same logic and move it to some other method?

Reply:
maybe default or check? what would you like to see here?

Reply:
i'm not sure what to name it offhand but, something that implies we're modifying it

Reply:
doing it inline in cluster.Create() would also probably be fine since it's relatively simple.

Reply:
ok I will look at this later in the week. Thanks for the feedback!